assessment-service/src/test/java/uk/gov/laa/ccms/caab/assessment/controller/AssessmentControllerTest.java

```diff
@@ -91,4 +92,50 @@ public void getAssessments() throws Exception {
     verify(assessmentService).getAssessments(criteria, names);
   }
 
+  @Test
+  public void deleteAssessments() throws Exception {
+    String providerId = "providerId";
+    String caseReferenceNumber = "caseReferenceNumber";
+    String status = "status";
+    List<String> names = new ArrayList<>(List.of("name"));
+
+    AssessmentDetail criteria = new AssessmentDetail()
+        .providerId(providerId)
+        .caseReferenceNumber(caseReferenceNumber)
+        .status(status);
+
+    this.mockMvc.perform(delete("/assessments")
+            .header("caab-User-Login-Id", "TestUser")
+            .param("provider-id", providerId)
+            .param("case-reference-number", caseReferenceNumber)
+            .param("name", String.join(",", names))
+            .param("status", status))
+        .andExpect(status().isNoContent());
+
+    verify(assessmentService).deleteAssessments(criteria, names);
+  }
+
+  @Test
+  public void deleteAssessments_returnsNoContentWhenNoMatchingAssessments() throws Exception {
```

Comment:
Already covered by the test above?

Reply:
These are the unit test just checking method calls, rather than actual data.
Required for the 80% coverage as integration test don't get included.